Dolphin scripts/Entrance Randomizer/lib/graph_creation.py

```diff
@@ -93,31 +105,84 @@ def create_vertices(
     return output_text
 
 
-def create_edges(transitions_map: Mapping[tuple[int, int], tuple[int, int]]):
-    connections = [(original[0], redirect[1]) for original, redirect in transitions_map.items()]
-    connections_two_way: list[tuple[int, int]] = []
-    connections_one_way: list[tuple[int, int]] = []
+def edge_text(
+    start: int,
+    end: int,
+    counter: int,
+    direct: Direction,
+    color: str | None,
+    line_type: LineType,
+):
+    direct_str = "true"
+    if direct == Direction.TWOWAY:
+        direct_str = "false"
+    output = (
+        f'<edge source="{transition_infos_dict_rando[start].area_id}" '
+        + f'target="{transition_infos_dict_rando[end].area_id}" isDirect="{direct_str}" '
+        + f'id="{counter}"'
+    )
+    if line_type == LineType.DASHED or color is not None:
+        output += ' ownStyles="{&quot;0&quot;:{'
+        if color is not None:
+            output += f"&quot;strokeStyle&quot;:&quot;{color}&quot;"
+            if line_type == LineType.DASHED:
+                output += ","
+        if line_type == LineType.DASHED:
+            output += "&quot;lineDash&quot;:&quot;2&quot;"
+        output += '}}"'
```

Comment:
added create_own_style function.
Also, `2` is how the website decides what type of lineDash to use. There are multiple to choose from. The one I'm using currently is desginated `2`, so that's why I'm using 2 here.